hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -985,14 +1009,17 @@ private void checkAuthenticationResponse(TcpClientConnection connection,
         }
         switch (authenticationStatus) {
             case AUTHENTICATED:
+                connectionProcessListener.authenticationSuccess();
```

Comment:
Don't we need extra information here? I mean, authentication succeeded, but succeeded to where? Same for the calls below

Reply:
Yes, I added the address parameter to all 3 methods.